source/lib-intro.tex

```diff
@@ -3715,8 +3714,7 @@
 Functions from the C standard library shall not throw exceptions%
 \indextext{specifications!C standard library exception}%
 \begin{footnote}
-That is, the C
-library functions can all be treated as if they
+That is, the C library functions can all be treated as if they
```

Comment:
```suggestion
That is, the C standard library functions can all be treated as if they
```
Use exactly the term we define in Clause 3, not a variant of it, per ISO rules.

Reply:
Good catch!